libraries.json

```diff
@@ -879,6 +903,13 @@
     "description": "Webpack is an open-source JavaScript module bundler. It is a module bundler primarily for JavaScript, but it can transform front-end assets like HTML, CSS, and images if the corresponding loaders are included.",
     "language": "JavaScript"
   },
+  "Wordpress": {
```

Comment:
I am not sure about WordPress. It is more like a platform and you have to learn how to use the software, not exactly a library that you use to write software.